src/combobox/ComboBox.ts

```diff
@@ -385,7 +387,7 @@ export default class ComboBox extends ComboBoxBase<ComboBoxProperties> {
 			clearable ? v('button', {
 				'aria-controls': menuId,
 				bind: this,
-				classes: this.classes(css.clear),
+				classes: this.classes(css.clear, iconCss.icon, iconCss.times),
```

Comment:
Is it really necessary to pass `icon` every time we pass a specific icon class, like `times`? Can't we write the CSS in a way where this isn't needed?